clients/filesystem-hadoop3/src/main/java/org/apache/gravitino/filesystem/hadoop/GravitinoVirtualFileSystemConfiguration.java

```diff
@@ -107,5 +112,46 @@ public class GravitinoVirtualFileSystemConfiguration {
   public static final long FS_GRAVITINO_FILESET_CACHE_EVICTION_MILLS_AFTER_ACCESS_DEFAULT =
       1000L * 60 * 60;
 
+  private static final Map<String, String> GVFS_S3_KEY_TO_HADOOP_KEY =
```

Comment:
If we add a new FS provider, do we need to update the GVFS code to make it work? If so, I think it is not a good design, we should also hide the client side configuration into each provider.

Reply:
In the long run, we can provide a method like `requiredPropertyKey` in the `FileSystemProvider` and load this provider by service provider. By doing so, we can register this property to the client dynamically.  Currently, we cannot avoid this hard coding.

Reply:
I still suggest to move this to each provider, we don't have to expose them here.

Reply:
The fact that this will make module `hadoop3-filesystem` depend on bundle jars in the runtime explicitly will hinder me from doing so. If this is not a big problem, I would like to do so.

Reply:
Why there is a dependency? If user sets the configurations via Hadoop `Configuration` in gvfs, when gvfs create a provider, it can pass the configs from gvfs' `Configuration` to provider's config map. Each provider can use this config map to check the configurations and initialize the FS.